src/libraries/Microsoft.Extensions.Configuration.Abstractions/README.md

```diff
@@ -0,0 +1,13 @@
+# Microsoft.Extensions.Configuration.Abstractions
```

Comment:
This file already added by @MSDN-WhiteKnight I guess

Reply:
Yes, in this commit: https://github.com/dotnet/runtime/pull/77733/files#diff-46f803b9b3f17e6cbf7cc2d229bd3e906e08ecf82eb7b2a21fe736cbb2637818 That's why this PR shows conflicts.